playbooks/common_examples/sonic_bgp.yaml

```diff
@@ -231,6 +254,15 @@
                         direction: in
                     route_reflector_client: true
                     route_server_client: true
+                    ip_afi:
+                      default_policy_name: bb
+                      send_default_route: true
+                    prefix_limit:
+                      max_prefixes: 100
+                      prevent_teardown: true
+                      warning_threshold: 80
```

Comment:
It would be good to also show "restart_timer" here.

Reply:
prevent_teardown and restart_timer cannot be enabled at the same time, there are other tasks in the playbook that show prevent_teardown as false and specify a restart_timer

Reply:
Thanks for explaining. I see the other example.